### ISSUE TITLE
Support for user-defined procedural collision?

### ISSUE THREAD

**Issue by UltraEngine** [jrouwe/JoltPhysics#704]
I was wondering if this library supports user-defined procedural collision shapes? This would work by defining a callback that accepts a bounding box and returns a list of triangles or polygons that can be collided within that bounding box. This is useful for custom terrain systems.

**Comment by jrouwe** [comment]
Yes it does, please see the `Creating Custom Shapes` section [here](https://jrouwe.github.io/JoltPhysics/).
